conda.recipe/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: dask
+  version: 'nightly'
+
+build:
+  number: 1
+
+source:
+  git_url: ../
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolz
+    - psutil
+    - chest
+    - heapdict
+    - readline
```

Comment:
I don't know. It was installed in my environment for some reason. But everything tests out fine with it removed.